src/Runners/PHPUnit/Runner.php

```diff
@@ -65,8 +65,6 @@ private function fillRunQueue(array $availableTokens): void
             && ($token = array_shift($availableTokens)) !== null
         ) {
             $executebleTest = array_shift($this->pending);
-            /** @psalm-suppress RedundantConditionGivenDocblockType **/
-            assert($executebleTest !== null);
```

Comment:
If you leave this here, what test is failing?

Reply:
Please look at  #539, I wrote there more info about problem

Reply:
Thank you for bringing this into my attention.
I've pushed to master the change, so now you cane rebase the PR onto it